NorwegianRefugeeCouncil/core
# [CORE24-296] fix(tests): Renamed jest config for e2e to avoid conflict

Jira ticket: JIRA-CORE24-296


## What have you implemented?

The test target searches for the default jest config file in each project, and since our e2e target were using that file for their configuration, the unit tests were picking it up and failing.
So I renamed the e2e config file, and the conflict is gone.

Also removed the exclusion in the CI configuration, as I realised that this was likely the issue there too.

## How to test it?

Run `npx nx run-many --target=test` and expect the unit test suites from all projects to run successfully. The api-e2e project should not be run as part of it, since it doesn't contain unit tests.

## Extra/next steps
